src/univariate/continuous/gamma.jl

```diff
@@ -1,25 +1,43 @@
 """
-    Gamma(α,θ)
+    Gamma <: ContinuousUnivariateDistribution
 
-The *Gamma distribution* with shape parameter `α` and scale `θ` has probability density
+The *gamma* probability distribution.
+
+# Constructors
+
+    Gamma(α|alpha|shape=1, θ|theta|scale=1)
+
+Construct a `Gamma` distribution object with shape `α` and scale `θ`.
+
+    Gamma(α|alpha|shape=1, β|beta|rate=1)
+
+Construct a `Gamma` distribution object with shape `α` and rate `β = 1/θ`.
+
+    Gamma(mean=, α|alpha|shape=)
+    Gamma(mean=, θ|theta|scale=)
+    Gamma(mean=, β|beta|rate=)
+    Gamma(mean=, std=)
+    Gamma(mean=, var=)
+
+Construct a `Gamma` distribution object matching the relevant moments and parameters.
+
+# Details
+A gamma distribution with shape parameter `α` and scale `θ` has probability density
 function
 
 ```math
 f(x; \\alpha, \\theta) = \\frac{x^{\\alpha-1} e^{-x/\\theta}}{\\Gamma(\\alpha) \\theta^\\alpha},
 \\quad x > 0
 ```
 
+# Examples
 ```julia
-Gamma()          # Gamma distribution with unit shape and unit scale, i.e. Gamma(1, 1)
-Gamma(α)         # Gamma distribution with shape α and unit scale, i.e. Gamma(α, 1)
-Gamma(α, θ)      # Gamma distribution with shape α and scale θ
-
-params(d)        # Get the parameters, i.e. (α, θ)
-shape(d)         # Get the shape parameter, i.e. α
```

Comment:
Wouldn't it be useful to keep mentioning this somewhere in the docstring?

Reply:
Probably, but not in examples (since this code doesn't actually run as-is, so isn't really a good example).

At the moment, I think the best option would be to have a `# Functions` section, which could be auto-generated.

Reply:
Good idea. Though since the information exists currently, maybe just mention these functions in a sentence, like "This distributions implements the following functions:"? Unless you want to write the auto-generation code, of course. ;-)

Reply:
I can, but in another PR (this is already getting obscenely complicated to keep up to date).